### DISCUSSION TITLE
ERR_HTTP_INVALID_STATUS_CODE from vercel now

### DISCUSSION THREAD

**Discussion by Nexxkinn** [vercel/vercel#4607] [Help] [answered]
Hello, I built a custom runtime for deno, and while I managed to properly run a bootsrap up until getting a 202 status after invoking a response to AWS Lambda, I got an RangeError exception from your library.

```
RangeError [ERR_HTTP_INVALID_STATUS_CODE] [ERR_HTTP_INVALID_STATUS_CODE]: Invalid status code: undefined
    at ServerResponse.writeHead (_http_server.js:248:11)
    at ServerResponse._implicitHeader (_http_server.js:239:8)
    at write_ (_http_outgoing.js:650:9)
    at ServerResponse.end (_http_outgoing.js:761:5)
    at DevServer.serveProjectAsNowV2 (/usr/local/lib/node_modules/now/dist/index.js:2:1243680)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at Server.DevServer.devServerHandler (/usr/local/lib/node_modules/now/dist/index.js:2:1238887) {
  code: 'ERR_HTTP_INVALID_STATUS_CODE'
}
```
Can someone explain to me what this error means and what did I miss?
You can look the build log [here](https://github.com/Nexxkinn/vercel-deno-runtime/runs/752741742?check_suite_focus=true) if you want to check it.

**Comment by Nexxkinn** [comment, marked as answer]
nvm figured it out




turns out i used a lowercase `statuscode` on my response, instead of camelCase `statusCode`

**Comment by TooTallNate** [comment]
Hey @Nexxkinn, just wanted to let you know that I also wrote a [Deno runtime](https://github.com/tootallnate/vercel-deno). Would love to collaborate!

**Reply by Nexxkinn** [reply]
Thanks, I'm currently developing it as a learning curve for both Vercel and Serverless framework. I'm also using parts of your runtime as a reference, since It's  simpler to read and understand the vercel runtime.

I'm currently on a way to emulate `NowRequest` and `NowResponse` on nodejs runtime and would glad to pull it to your runtime whenever possible 🥂
